app/lib/emoji.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'singleton'
+
+class Emoji
+  include Singleton
+
+  def initialize
+    data = Oj.load(File.open(File.join(Rails.root, 'node_modules', 'emojione', 'emoji.json')))
```

Comment:
this may be a heretic idea, but what about we simply copy the json from the package somewhere else in the codebase and update it manually or with some script from time to time?